CRITICAL - 2022-12-29 09:08:37 --> Undefined variable $validation
in APPPATH\Views\view_token.php on line 111.
 1 APPPATH\Views\view_token.php(111): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $validation', 'APPPATH\\Views\\view_token.php', 111)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\view_token.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('view_token', [], true)
 5 APPPATH\Controllers\Register.php(40): view('view_token')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Register->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-29 09:21:09 --> Undefined variable $validation
in APPPATH\Views\view_token.php on line 111.
 1 APPPATH\Views\view_token.php(111): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $validation', 'APPPATH\\Views\\view_token.php', 111)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\view_token.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('view_token', [], true)
 5 APPPATH\Controllers\Register.php(41): view('view_token')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Register->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-29 09:29:22 --> Undefined variable $validation
in APPPATH\Views\view_token.php on line 111.
 1 APPPATH\Views\view_token.php(111): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $validation', 'APPPATH\\Views\\view_token.php', 111)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\view_token.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('view_token', [], true)
 5 APPPATH\Controllers\Register.php(41): view('view_token')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Register->token()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-29 09:48:23 --> Undefined variable $validation
in APPPATH\Views\view_token.php on line 111.
 1 APPPATH\Views\view_token.php(111): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $validation', 'APPPATH\\Views\\view_token.php', 111)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\view_token.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('view_token', [], true)
 5 APPPATH\Controllers\Register.php(41): view('view_token')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Register->token()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
CRITICAL - 2022-12-29 09:52:14 --> Undefined variable $validation
in APPPATH\Views\view_token.php on line 111.
 1 APPPATH\Views\view_token.php(111): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $validation', 'APPPATH\\Views\\view_token.php', 111)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\view_token.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('view_token', [], true)
 5 APPPATH\Controllers\Register.php(49): view('view_token')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Register->token()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-29 10:07:27 --> mysqli_sql_exception: Duplicate entry '' for key 'username' in C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\barberhouse\app\Controllers\Auth.php(61): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\Auth->register()
#9 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\barberhouse\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-12-29 10:07:27 --> Duplicate entry '' for key 'username'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tbl_user` (`email`, `id_role`, `token`) VALUES (\'a@gmail.com\', \'1\', \'ZFxfVW\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tbl_user` (`email`, `id_role`, `token`) VALUES (\'a@gmail.com\', \'1\', \'ZFxfVW\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tbl_user` (`email`, `id_role`, `token`) VALUES (\'a@gmail.com\', \'1\', \'ZFxfVW\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbl_user` (`email`, `id_role`, `token`) VALUES (:email:, :id_role:, :token:)', [...], false)
 5 SYSTEMPATH\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(749): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(667): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Auth.php(61): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Auth->register()
10 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
11 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
ERROR - 2022-12-29 10:09:42 --> mysqli_sql_exception: Duplicate entry '' for key 'username' in C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\barberhouse\app\Controllers\Auth.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\Auth->register()
#9 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\barberhouse\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-12-29 10:10:34 --> mysqli_sql_exception: Duplicate entry '' for key 'username' in C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\barberhouse\app\Controllers\Auth.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\Auth->register()
#9 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\barberhouse\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-12-29 10:10:59 --> mysqli_sql_exception: Duplicate entry '' for key 'username' in C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\barberhouse\app\Controllers\Auth.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\Auth->register()
#9 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\barberhouse\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-12-29 10:14:03 --> mysqli_sql_exception: Column 'username' cannot be null in C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\barberhouse\app\Controllers\Auth.php(63): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\Auth->register()
#9 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\barberhouse\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-12-29 10:15:18 --> mysqli_sql_exception: Column 'username' cannot be null in C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\barberhouse\app\Controllers\Auth.php(62): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\Auth->register()
#9 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\barberhouse\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\barberhouse\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-12-29 10:22:27 --> count(): Argument #1 ($value) must be of type Countable|array, null given
in APPPATH\Controllers\Register.php on line 39.
 1 APPPATH\Controllers\Register.php(39): count(null)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Register->token()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
